jerry-core/api/jerry.c

```diff
@@ -4066,41 +4067,31 @@ jerry_from_property_descriptor (const jerry_property_descriptor_t *src_prop_desc
 } /* jerry_from_property_descriptor */
 
 /**
-* ToPropertyDescriptor abstract operation.
-*
-* @return thrown error - if any conversion error happens
-*        ECMA_VALUE_TRUE - otherwise.
-*/
-bool
+ * ToPropertyDescriptor abstract operation.
+ *
+ * @return true - if the conversion is successful
+ *         thrown error - otherwise
```

Comment:
This description is not the same as in the docs. We should keep it the same.

Reply:
This is not the same in other cases as well. The doc is more verbose (intentionally).